src/main/fc/rc_adjustments.c

```diff
@@ -214,7 +220,27 @@ static const adjustmentConfig_t defaultAdjustmentConfigs[ADJUSTMENT_FUNCTION_COU
         .adjustmentFunction = ADJUSTMENT_HORIZON_STRENGTH,
         .mode = ADJUSTMENT_MODE_SELECT,
         .data = { .switchPositions = 255 }
+    },
+#ifdef USE_OSD_PROFILES
```

Comment:
This code has to be removed - underlying problem is to be fixed

Reply:
This code will handle correct array position automatically:
```
[ADJUSTMENT_OSD_PROFILE - ADJUSTMENT_FUNCTION_CONFIG_INDEX_OFFSET] = {
        .adjustmentFunction = ADJUSTMENT_OSD_PROFILE,
        .mode = ADJUSTMENT_MODE_SELECT,
        .data = { .switchPositions = 4 }
    }
```

Also there requirement for matching array position was possibly  removed with recent adjustment optimization PR's